edb/lib/std/30-jsonfuncs.edgeql

```diff
@@ -85,6 +85,31 @@ std::json_get(
     $$;
 };
 
+
+CREATE FUNCTION
```

Comment:
We want this because it produces a better error message than leaving it to the cast, or what?

Reply:
This function lets us produce different error messages for when a tuple is missing an element and when the element is there but explicitly set to null.